plugins/services/src/js/components/modals/ServiceRootGroupModal/index.tsx

```diff
@@ -290,7 +290,26 @@ class ServiceRootGroupModal extends React.Component<
           isPending: false
         });
         return;
+      case "Overcommit":
+        this.setState({
+          errors: errorsFromOvercommitData(data),
+          isPending: false,
+          isForce: true
+        });
+        return;
       default:
+        if (mesos) {
+          this.setState({
+            errors: {
+              form: [
+                <Trans key="quotaError">
+                  Unable to create group's quota: {message}
+                </Trans>
+              ]
+            },
+            isPending: false
+          });
+        }
         this.setState({
```

Comment:
@TattdCodeMonkey , should this be an else?
So when it's mesos (which I think can be renamed to mesosError) we say "Unable to create group's quota: ", otherwise we say "Unable to create group: ".
This is also the reason the integration test is failing.
![Screenshot from 2019-08-16 10-43-26](https://user-images.githubusercontent.com/40791275/63152475-9dc01180-c014-11e9-99e4-9cfea0e3f912.png)

Reply:
Yes there should be an else or return inside the if 🤦‍♂️